### PR TITLE
Remove mentions of Addressbook

### LINKED ISSUE #151: [PE-D] Incorrect message: trace of address book
When `add` is entered, the message shows address book

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: Ruofan1023/ped#1

### PR DESCRIPTION
**Pull request by tchiong**
Removed all mentions of Address book where relevant.
Fixes #151

### CODE REVIEW

File: src/test/java/seedu/placebook/model/ModelManagerTest.java [hunk 1]
```diff
@@ -97,9 +97,9 @@ public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException
 
     @Test
     public void equals() {
-        AddressBook addressBook = new AddressBookBuilder().withPerson(ALICE).withPerson(BENSON).build();
+        Contacts addressBook = new AddressBookBuilder().withPerson(ALICE).withPerson(BENSON).build();
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/model/ModelManagerTest.java:100]: will we have to change this to contacs = new ContactBuilder() etc?

File: src/test/java/seedu/placebook/model/ModelManagerTest.java [hunk 2]
```diff
@@ -97,9 +97,9 @@ public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException
...
         Schedule schedule = getTypicalSchedule();
-        AddressBook differentAddressBook = new AddressBook();
+        Contacts differentAddressBook = new Contacts();
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/model/ModelManagerTest.java:102]: same as above

File: src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java [hunk 3]
```diff
@@ -63,26 +63,26 @@ public void readAddressBook_invalidAndValidPersonAddressBook_throwDataConversion
     @Test
     public void readAndSaveAddressBook_allInOrder_success() throws Exception {
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java:64]: method names like this should change also i think

File: src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java [hunk 4]
```diff
@@ -63,26 +63,26 @@ public void readAddressBook_invalidAndValidPersonAddressBook_throwDataConversion
...
         Path filePath = testFolder.resolve("TempAddressBook.json");
-        AddressBook original = getTypicalAddressBook();
-        JsonAddressBookStorage jsonAddressBookStorage = new JsonAddressBookStorage(filePath);
+        Contacts original = getTypicalAddressBook();
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java:66]: getTypicalContacts()?

File: src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java [hunk 5]
```diff
@@ -63,26 +63,26 @@ public void readAddressBook_invalidAndValidPersonAddressBook_throwDataConversion
...
+        JsonContactsStorage jsonAddressBookStorage = new JsonContactsStorage(filePath);
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/storage/JsonContactsStorageTest.java:67]: variable name change to jsonContactsStorage?

File: src/test/java/seedu/placebook/storage/StorageManagerTest.java [hunk 6]
```diff
@@ -27,7 +27,7 @@
 
     @BeforeEach
     public void setUp() {
-        JsonAddressBookStorage addressBookStorage = new JsonAddressBookStorage(getTempFilePath("ab"));
+        JsonContactsStorage addressBookStorage = new JsonContactsStorage(getTempFilePath("ab"));
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/storage/StorageManagerTest.java:30]: here also

File: src/test/java/seedu/placebook/storage/StorageManagerTest.java [hunk 7]
```diff
@@ -58,15 +58,15 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
          */
-        AddressBook original = getTypicalAddressBook();
-        storageManager.saveAddressBook(original);
-        ReadOnlyAddressBook retrieved = storageManager.readAddressBook().get();
-        assertEquals(original, new AddressBook(retrieved));
+        Contacts original = getTypicalAddressBook();
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/storage/StorageManagerTest.java:61]: getTypicalContacts()

File: src/test/java/seedu/placebook/testutil/AddressBookBuilder.java [hunk 8]
```diff
@@ -10,13 +10,13 @@
  */
 public class AddressBookBuilder {
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/testutil/AddressBookBuilder.java:11]: change to ContactsBuilder?

File: src/test/java/seedu/placebook/testutil/TypicalPersons.java [hunk 9]
```diff
@@ -62,8 +62,8 @@ private TypicalPersons() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the typical persons.
      */
-    public static AddressBook getTypicalAddressBook() {
-        AddressBook ab = new AddressBook();
+    public static Contacts getTypicalAddressBook() {
```
Comment (NecroWolf28) [inline src/test/java/seedu/placebook/testutil/TypicalPersons.java:65]: getTypicalContacts()

**Review by YaleChen299 — APPROVED** [review]
LGTM

**Pull request merged**